log/config.go

```diff
@@ -0,0 +1,18 @@
+package log
+
+// Config is the confuration struct for the log package.
+//
+// Can be deserialized from YAML.
+type Config struct {
+	// Level is the log level you want to set your service to.
+	Level Level
+}
+
+// InitFromConfig initializes the log package using the given Config.
+func InitFromConfig(cfg Config) {
+	if cfg.Level == "" {
+		cfg.Level = InfoLevel
+	}
+	level := cfg.Level
+	InitLogger(level)
```

Comment:
I just merged the other PR so you need to rebase on master and use `InitLoggerJSON` here instead 😅